db/db_impl/db_impl.h

```diff
@@ -2287,6 +2287,7 @@ class DBImpl : public DB {
   // In 2PC these are the writes at Prepare phase.
   const bool two_write_queues_;
   const bool manual_wal_flush_;
+  const CompressionType wal_compression_;
```

Comment:
I don't think this is needed because it's part of `immutable_db_options_`

Reply:
Addressed.